vips/vip-440/bscmainnet.ts

```diff
@@ -0,0 +1,197 @@
+import { BigNumberish } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+import { NORMAL_TIMELOCK } from "src/vip-framework";
+
+const { VTREASURY, RESILIENT_ORACLE, REDSTONE_ORACLE, UNITROLLER, ACCESS_CONTROL_MANAGER } =
+  NETWORK_ADDRESSES.bscmainnet;
+export const PROTOCOL_SHARE_RESERVE = "0xCa01D5A9A248a830E9D93231e791B1afFed7c446";
+export const THE_REDSTONE_FEED = "0xFB1267A29C0aa19daae4a483ea895862A69e4AA5";
+const THE_MAX_STALE_PERIOD = 30 * 60; // 30 minutes
+const REDUCE_RESERVES_BLOCK_DELTA = "28800";
+
+const USDT = "0x55d398326f99059fF775485246999027B3197955";
+const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+const BTCB = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+const RISK_FUND_CONVERTER = "0xA5622D276CcbB8d9BBE3D1ffd1BB11a0032E53F0";
+const USDT_PRIME_CONVERTER = "0xD9f101AA67F3D72662609a2703387242452078C3";
+const USDC_PRIME_CONVERTER = "0xa758c9C215B6c4198F0a0e3FA46395Fa15Db691b";
+const BTCB_PRIME_CONVERTER = "0xE8CeAa79f082768f99266dFd208d665d2Dd18f53";
+const ETH_PRIME_CONVERTER = "0xca430B8A97Ea918fF634162acb0b731445B8195E";
+const XVS_VAULT_CONVERTER = "0xd5b9AE835F4C59272032B3B954417179573331E0";
+
+export const CONVERSION_INCENTIVE = 1e14;
+export const converterBaseAssets = {
+  [RISK_FUND_CONVERTER]: USDT,
+  [USDT_PRIME_CONVERTER]: USDT,
+  [USDC_PRIME_CONVERTER]: USDC,
+  [BTCB_PRIME_CONVERTER]: BTCB,
+  [ETH_PRIME_CONVERTER]: ETH,
+  [XVS_VAULT_CONVERTER]: XVS,
+};
+
+export const marketSpec = {
+  vToken: {
+    address: "0x86e06EAfa6A1eA631Eab51DE500E3D474933739f",
+    name: "Venus THE",
+    symbol: "vTHE",
+    underlying: {
+      address: "0xF4C8E32EaDEC4BFe97E0F595AdD0f4450a863a11",
+      decimals: 18,
+      symbol: "THE",
+    },
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: UNITROLLER,
+    isLegacyPool: true,
+  },
+  interestRateModel: {
+    address: "0x77C50b4d5d8b84511a4787aaF5BC22E5A496B1Ef",
+    base: "0",
+    multiplier: "0.09",
+    jump: "3.5",
+    kink: "0.45",
+  },
+  initialSupply: {
+    amount: parseUnits("12286", 18),
+    vTokenReceiver: VTREASURY,
```

Comment:
The received on mainnet will finally be `0x1c6C2498854662FDeadbC4F14eA2f30ca305104b`